### ISSUE TITLE
Root CMakeLists forces visual studio compiler to 2012/v110

### ISSUE THREAD

**Issue by baldurk** [KhronosGroup/glslang#12]
I noticed while changing the CMakeLists.txt files for the osinclude pull request that there is this line in the root CMakeLists.txt:

```cmake
    set(CMAKE_GENERATOR_TOOLSET "v110" CACHE STRING "Platform Toolset" FORCE)
```

I'm not sure why this toolset is important (is it a minimum, since C++11 features are used that don't work in v100? or is using a higher toolset undesirable?), but it means a bit of fiddling locally if you don't have 2012 installed.

Perhaps there is a way to check in CMake if that toolset exists before forcing it?

**Comment by johnkslang** [comment]
It is required to make binaries that execute on Windows XP (seems somewhat artificial).  At some point we could say that's too old, or make it conditional on the version of MSVC.

**Comment by johnkslang** [comment]
This support was removed (fixed) by 5d60ca5744ee1eb64ff134e635329a0ef2cff835.
